doc/bundle.md

```diff
@@ -0,0 +1,119 @@
+@function steal-tools.bundle bundle
+@parent steal-tools.JS 
+
+Similar to [steal-tools.build] it creates a bundle of modules and its dependencies but a glob pattern
+can be provided to filter out module during the build.
+
+@signature `stealTools.bundle(config, options)`
+
+@param {steal-tools.StealConfig} config 
+Specifies configuration values to set on Steal's loader.
+  
+@param {steal-tools.BuildOptions} [options]
+Specifies the behavior of the build.
+  
+@return {Promise<steal-tools.BuildResult} A Promise that resolves when the build is complete.
+
+@body
+
+## Use
+
+The following uses steal-tool's `bundle` method to programatically build out the "my-app"
+dependencies in the `node_modules` folder.
+
+    var path = require("path");
+    var stealTools = require("steal-tools");
+
+    var promise = stealTools.bundle({
+      config: path.join(__dirname, "package.json!npm")
+    }, {
+      filter: "node_modules/**/*"
+    });
```

Comment:
You might want to use backticks for these codeblocks. At least GitHub has trouble thinking that the asterisks are not for bolding...